README.md

```diff
@@ -18,7 +15,7 @@ version: '3'
 
 services:
     db-master:
-        image: caffeina/mariadb-replication
```

Comment:
Hi @db260179,

why force users to build the image when they can use the one already built?